### PR TITLE
Adding doc template to getIntensityPeaks function

### PR DESCRIPTION
**Pull request by Daniel-Tojal**
Inputs and Outputs descriptions are still to be determined. Possibly a better description?

### CODE REVIEW

File: src/Functions/getIntensityPeaks.m [hunk 1]
```diff
@@ -1,12 +1,34 @@
...
+% update to support electrodes with more ring contacts)!
+%
+% USAGE:
+%
+%    [peakLocs, peakWaveCenters, peakValues, threshIntensityProfile, 
+%    threshold, contactAreaCenter, contactAreaWidth, xrayMarkerAreaCenter, xrayMarkerAreaWidth] 
+%    = getIntensityPeaks(intensityProfile, skelScaleMm, filterIdxs)
+%
+% INPUTS: 
+%    intensityProfile:
```
Comment (Daniel-Tojal) [inline src/Functions/getIntensityPeaks.m]: @adhusch Could you please provide a description for the inputs and outputs?

File: src/Functions/getIntensityPeaks.m [hunk 2]
```diff
@@ -1,12 +1,34 @@
-%% Get peaks respectivly center points at threshold below the peaks from 1D
-% intensity profile. Expected peakWaveCenter number == 4 for now (needs update to support electrodes with more ring contacts)!
-%
-% Andreas Husch
-% Centre Hospitalier de Luxembourg, Dep. of Neurosurgery /
-% University of Luxembourg - Luxembourg Centre for Systems Biomedicne
-% 2014 - 2017
-% mail@andreashusch.de, husch.andreas@chl.lu
 function [peakLocs, peakWaveCenters, peakValues, threshIntensityProfile, threshold, contactAreaCenter, contactAreaWidth, xrayMarkerAreaCenter, xrayMarkerAreaWidth] = getIntensityPeaks(intensityProfile, skelScaleMm, filterIdxs)
+% Get peaks, respectively center points, at threshold below the peaks from 1D
```
Comment (Daniel-Tojal) [inline src/Functions/getIntensityPeaks.m:2]: @adhusch I always take your file header to put in the function description as it suits it quite well, but in this case this header appears to be more akin to a comment than a description. Maybe you could provide a more concise description, or tell me what i should edit out/edit in?

Comment (adhusch) [reply src/Functions/getIntensityPeaks.m:2]: Well, that function is a clean code disaster anyway, has way to many return values ;) Should be refactored in total..

Comment (laurentheirendt) [reply src/Functions/getIntensityPeaks.m:2]: Before refactoring this functions, I suggest to write a test for it first. @loicmarx ?

@adhusch, may you provide @Daniel-Tojal the description of the keywords for now?

Comment (adhusch) [reply src/Functions/getIntensityPeaks.m:2]: peakLocs 1d peak locations, 
peakWaveCenters 1d peak wave center locations
peakValues peak values 
threshIntensityProfile thresholded intensity profile 
threshold theshold
contactAreaCenter center of the assumed contact area
contactAreaWidth with of the assumed contact area
 xrayMarkerAreaCenter center of the xray marker (if any)
 xrayMarkerAreaWidth width of the xray maker (if any)

**Pull request merged**